tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TaskMapper.java

```diff
@@ -213,18 +214,43 @@ private Double convertMemoryUnitTypeToGiB(String memoryUnit) {
         return GIB_TO_GIB;
     }
 
-    public TesTask mapToTesTask(PipelineRun run) {
-        return TesTask.builder()
-                .id(String.valueOf(run.getId()))
-                .name(run.getPodId())
-                .resources(createTesResources(run))
-                .executors(createListExecutor(run))
-                .inputs(createTesInput(ListUtils.emptyIfNull(run.getPipelineRunParameters())))
-                .outputs(createTesOutput(ListUtils.emptyIfNull(run.getPipelineRunParameters())))
-                .creationTime(run.getStartDate().toString())
-                .logs(createTesTaskLog(run.getId()))
-                .state(createTesState(run))
-                .build();
+    public TesTask mapToTesTask(PipelineRun run, TaskView view) {
+        return filterTesTaskWithView(run, view);
+    }
+
+    private TesTask filterTesTaskWithView(PipelineRun run, TaskView view) {
+        switch (view) {
```

Comment:
what about to make incremental building of TesTask. 
As far as i know basic mode fully includes minamal mode and full mode includes basic.
So something like this should be good:
```
final TesTaskBuilder taskBuilder = TesTask.builder()
                                                                  .id(String.valueOf(run.getId()))
                                                                  .state(createTesState(run));
if (view == MINIMAL) {
     return taskBuilder.build();
}
taskBuilder.
        .name(run.getPodId())
                        .resources(createTesResources(run))
                        .executors(createListExecutor(run))
                        .outputs(createTesOutput(ListUtils.emptyIfNull(run.getPipelineRunParameters())))
                        .creationTime(run.getStartDate().toString())

if (view == BASIC) {
     return taskBuilder.build();
}
....
```